packages/paypal-utils/src/PaypalCommerceCreditBanner.test.tsx

```diff
@@ -20,6 +20,8 @@ describe('PaypalCommerceCreditBanner', () => {
     const checkoutState = checkoutService.getState();
     const defaultProps = {
         onUnhandledError: jest.fn(),
+        containerId: 'paypal-commerce-credit-banner-container',
```

Comment:
We need to clone all changes for PPCP to BCP

Reply:
It will be done in separate ticket